### ISSUE TITLE
traceId: 1 to 32 character lower-hex string

### ISSUE THREAD

**Issue by BPing** [sfhardman/kong-plugin-zipkin#6]
Cannot decode spans due to IllegalArgumentException(VYc0IZ1GWo2xxJpz5ZufQK6TiNERv2pI should be a 1 to 32 character lower-hex string with no prefix reading List<Span> from json)

**Comment by sfhardman** [comment]
Can you provide more info for troubleshooting please?  e.g. which version of Zipkin and Kong?  Which scenario (Simulate server, simulate client, simulate client and server)?

**Comment by BPing** [comment]
I use the latest version of the Zipkin docker image, but I use the V1 version of the interface。
Only accept 1 to 32 character lower-hex string 

`original code`
```
local function random_string_of_len(length)
  if length > 32 then
    ngx.log(ngx.ERROR, "maximum random_string_of_len length exceeded", length)
    -- not sure how to propagate the error....
    return nil
  end
  return string.sub(utils.random_string(), 1, length)
end
```

i change `string.sub(utils.random_string(), 1, length)` to `string.sub(hexstr.to_hex(utils.random_string()), 1, length)`，then ，it can run normally

**Comment by sfhardman** [comment]
Would you like to submit a pull request with a test?  If not I will have a look at it.

**Comment by sfhardman** [comment]
Fixed in 0.1.1, thanks for reporting the issue
